src/main/java/com/zipcodewilmington/phone/PhoneNumberFactory.java

```diff
@@ -20,14 +24,33 @@ private PhoneNumberFactory() {
      * @return array of randomly generated PhoneNumber objects
      */ //TODO - Implement logic
     public static PhoneNumber[] createRandomPhoneNumberArray(int phoneNumberCount) {
-        return null;
+        PhoneNumber[] phoneNumberArray = new PhoneNumber[phoneNumberCount];
+
+        for (int i=0; i<phoneNumberCount; i++)
+        {
+            phoneNumberArray[i]=createRandomPhoneNumber();
+        }
+
+        return phoneNumberArray;
     }
 
     /**
      * @return an instance of PhoneNumber with randomly generated phone number value
      */ //TODO - Implement logic
     private static PhoneNumber createRandomPhoneNumber() {
-        return createPhoneNumberSafely(-1, -1, -1);
```

Comment:
I provided you all with a `RandomNumberFactory` class in the `tools` package to clean this up...
```java
int areaCode = RandomNumberFactory.createInteger(100, 999);
int centralOfficeCode = RandomNumberFactory.createInteger(100, 999);
int phoneLineCode = RandomNumberFactory.createInteger(1000, 9999);
```